commit 9ee3e8ca2114a013f0afd9ea4323a51bfa3ef238
Author:     Isaac Yang <isaacy@nvidia.com>
AuthorDate: Mon Jun 1 11:58:08 2026 -0700
Commit:     GitHub <noreply@github.com>
CommitDate: Mon Jun 1 18:58:08 2026 +0000

    [main] Cherry-pick from [2.8]: Lower a log level on one specific message (#4674) (#4731)
    
    ### Description
    
    Forward-port 2.8 commit `899b872d519a5e743aee021d249915f0659fa55b` to
    `main`.
    
    This lowers the severity of one server engine log message.
    
    ### Types of changes
    - [x] Non-breaking change (fix or new feature that would not break
    existing functionality).
    - [ ] Breaking change (fix or new feature that would cause existing
    functionality to change).
    - [ ] New tests added to cover the changes.
    - [ ] Quick tests passed locally by running `./runtest.sh`.
    - [ ] In-line docstrings updated.
    - [ ] Documentation updated.
    
    ### Verification
    
    - `PYENV_VERSION=3.10.13 python3 -m pytest -q
    tests/unit_test/private/fed/server/server_engine_test.py`
    - `PYENV_VERSION=3.10.13 ./runtest.sh -s`
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index abe3c064..56011c32 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -851,7 +851,7 @@ class ServerEngine(ServerEngineInternalSpec, StreamableEngine):
         data = return_data.payload
         clients = data.get(ServerCommandKey.CLIENTS, None)
         if clients is None:
-            self.logger.error(f"parent failed to return clients info for job {job_id}")
+            self.logger.debug(f"parent failed to return clients info for job {job_id}")
         return clients
 
     def update_job_run_status(self):
